scripts/size-limit/index.js

```diff
@@ -0,0 +1,90 @@
+import path from "path"
+import fs from "fs"
+import sizeLimit from "size-limit"
+import filePlugin from "@size-limit/file"
+import esbuildPlugin from "@size-limit/esbuild"
+import bytes from "bytes-iec"
+
+import * as funcs from "../../dist/index.cjs"
+
+const formatBytes = (size) => {
+  return bytes.format(size, { unitSeparator: " " })
```

Comment:
Hey @aleksandrjet. Nice addition to the docs. A minor addition to make it less dependent on third-party libraries:

```
const bytesFormatter = Intl.NumberFormat("en", {
  notation: "compact",
  style: "unit",
  unit: "byte",
  unitDisplay: "narrow",
})

console.log(bytesFormatter.format(5351)) // "5.4KB"
```

Feel free to delete the comment if it's off-topic.

P.S. I am not the developer of Tempo.

Reply:
This is a good addition, but I see a small problem with the native format display due to the missing space, which I was not able to add through the config

In my opinion, the option with `bytes-iec` is read better. Also `bytes-iec` is used in `size-limit` which is already present in the dependencies

Intl.NumberFormat
![format_native](https://github.com/formkit/tempo/assets/79018733/d247e249-d6a2-4bcc-be5c-a7634c6cf857)

bytes-iec
![format_bytes_iec](https://github.com/formkit/tempo/assets/79018733/9519d874-b6c4-4b73-a6cb-e193bddca570)

Reply:
In this case you could use `.replace()` or `.formatToParts()` to add a space, but that would complicate the code and your solution with `bytes-iec` looks more readable. Good point. 👍🏻